DevOps/Vagrantfile

```diff
@@ -0,0 +1,34 @@
+require 'vagrant-openstack-provider'
+Vagrant.configure('2') do |config|
+  config.ssh.username = 'ubuntu'
+  config.ssh.private_key_path = "C:\\Users\\Administrator\\Desktop\\Pardeclaves\\wincmmv.pem"
```

Comment:
You need to use an environment variable for this path.

Reply:
Variable OS_PRIVATE_KEY_PATH is created on environment file.